Makefile

```diff
@@ -49,4 +54,27 @@ sonar:
 
 .PHONY: sonar-pr-analysis
 sonar-pr-analysis:
-	mvn sonar:sonar -P sonar-pr-analysis
\ No newline at end of file
+	mvn sonar:sonar -P sonar-pr-analysis
+
+# Docker build, tag, push and run targets replacing zip based packaging
+.PHONY: docker-build
+docker-build:
+	docker build -t $(docker_repository):$(version) .
+
+.PHONY: docker-tag
```

Comment:
I'd remove both `docker-tag` and `docker-push`. Concourse won't use these targets and we don't want developers tagging and pushing images either

Reply:
Yeah, that's a fair shout. I'd been thinking the same thing.
Done.